.github/dependabot.yml

```diff
@@ -7,6 +7,25 @@ updates:
       day: tuesday
     reviewers:
       - "mongodb/mongocli"
+    groups:
+      internal-dependencies:
+          patterns:
+            - "go.mongodb.org*" # A single dependency name
```

Comment:
Those should have the highest priority and can be labeled differently for help rota actions

Reply:
this will include the go driver which is kind of 🤷 for us

Reply:
Removing section. It will introduce more harm than good.